### PR TITLE
Fix flannel issues

### PR DESCRIPTION
**Pull request by ajeddeloh**
Fix https://github.com/coreos/bugs/issues/1439 and https://github.com/coreos/bugs/issues/1436.

### CODE REVIEW

File: app-admin/flannel/files/flanneld-rkt.service [hunk 1]
```diff
@@ -27,10 +27,15 @@ ExecStart=/usr/bin/rkt run --net=host \
...
    --volume ssl,kind=host,source=${ETCD_SSL_DIR},readOnly=true \
+   --volume certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+   --volume resolv,kind=host,source=/etc/resolv.conf,readOnly=true \
    --mount volume=runsystemd,target=/run/systemd \
    --mount volume=runflannel,target=/run/flannel \
    --mount volume=ssl,target=${ETCD_SSL_DIR} \
+   --mount volume=certs,target=/etc/ssl/certs \
+   --mount volume=resolv,target=/etc/resolv.conf \
    ${FLANNEL_IMG}:${FLANNEL_VER} \
+   --exec /opt/bin/flanneld \
```
Comment (crawford) [inline app-admin/flannel/files/flanneld-rkt.service]: Why is this change needed? We were running flanneld within a shell when we used Docker.

Comment (ajeddeloh) [inline app-admin/flannel/files/flanneld-rkt.service]: I don't think we were. In the early-docker version we run it directly with /opt/bin/flanneld as the command (see below)

`ExecStart=/usr/libexec/sdnotify-proxy /run/flannel/sd.sock \
  /usr/bin/docker run --net=host --privileged=true --rm \
  --volume=/run/flannel:/run/flannel \
  --env=NOTIFY_SOCKET=/run/flannel/sd.sock \
  --env=AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
  --env=AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
  --env-file=${FLANNEL_ENV_FILE} \
  --volume=/usr/share/ca-certificates:/etc/ssl/certs:ro \
  --volume=${ETCD_SSL_DIR}:${ETCD_SSL_DIR}:ro \
${FLANNEL_IMG}:${FLANNEL_VER} /opt/bin/flanneld --ip-masq=true`

Comment (crawford) [inline app-admin/flannel/files/flanneld-rkt.service]: Oh yeah, you're right. That's odd that we wrapped that in a shell. Maybe @eyakubovich remembers why that was done.

Comment (eyakubovich) [inline app-admin/flannel/files/flanneld-rkt.service]: I don't remember wrapping anything in a shell. Is it the .aci that has it wrapped in a shell?

Comment (ajeddeloh) [inline app-admin/flannel/files/flanneld-rkt.service]: yes. From cat-manifest on the aci hosted on quay:
```
...
  "app": {
    "exec": [
      "/bin/sh",
      "-c",
      "/opt/bin/flanneld"
    ],  
...
```

**Comment by crawford** [comment]
LGTM
